src/core/runtime.d

```diff
@@ -328,45 +390,78 @@ struct Runtime
      *
      *             if (fp)
      *             {
+     *                 ++result.executed;
      *                 try
      *                 {
      *                     fp();
+     *                     ++result.passed;
      *                 }
      *                 catch (Throwable e)
      *                 {
      *                     writeln(e);
-     *                     failed++;
      *                 }
      *             }
      *         }
      *     }
-     *     return failed == 0;
+     *     if(result.executed != result.passed)
+     *         result.summarize = true; // print failure
+     *     else
+     *         result.runMain = true; // all UT passed
+     *     return result;
      * }
      * ---------
      */
+    static @property void extModuleUnitTester( ExtendedModuleUnitTester h )
```

Comment:
I would prefer that the name, `ext`, was not shortened. Most likely you only going to call this method once, there's no reason to make it less clear to save a few letters.

Reply:
I wished to call it `moduleUnitTester`, but since there is an accessor, I'm reluctant to do that (the previous version of this PR did that, but the function pointer was returning a value compatible with bool).

I can change this.